clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -75,11 +89,22 @@ class AzureResourceManagerClient extends AzureBaseClient {
     }
   }
 
-  public boolean resourceGroupExists(AzureCredentials creds, String resourceGroupName) {
-    return this.getResourceManagementClient(creds).getResourceGroupsOperations().checkExistence(resourceGroupName).isExists()
+  ArrayList<ResourceGroup> getResourcesGroupsForApp(AzureCredentials creds, String applicationName) {
```

Comment:
Minor, but `List<ResourceGroup>` or `Collection<ResourceGroup>` would be more idiomatic.